public/js/index.js

```diff
@@ -551,10 +551,10 @@ Index.handleContextMenu = function (option, id) {
         });
         break;
     case "read":
-        LRR.openInNewTab(`./reader?id=${id}`);
+        LRR.openInNewTab(new LRR.apiURL(`/reader?id=${id}`).toString());
```

Comment:
Is the `toString()` required here? I haven't been seeing it in other uses of the class like with fetch and the like

Reply:
The API documentation for window.open is ambiguous, but a quick test suggests that it is not required.

Also caught some more URL references I'd missed.